project1/server/file_handler/my_file_handler.h

```diff
@@ -0,0 +1,50 @@
+//
+// Created by jake on 1/25/21.
```

Comment:
Can we put a proper `@file` block here instead? See other `.h` files for examples.